plugins/git/hm-module.nix

```diff
@@ -0,0 +1,24 @@
+{
+  config = _: _: {
+    programs.yazi = {
+      initLua = ''
+        require("git"):setup()
```

Comment:
change it to use require option
see here: https://github.com/lordkekz/nix-yazi-plugins/blob/ee2026bbdde9df21112293f4a36da848e213d7ca/plugins/starship/hm-module.nix#L3